Sources/Profile/Snapshot/ProfileSnapshot.swift

```diff
@@ -25,7 +25,7 @@ public struct ProfileSnapshot:
 
 	/// Settings for this profile in the app, contains default security configs
 	/// as well as display settings.
-	public let appPreferences: AppPreferences
+	public private(set) var appPreferences: AppPreferences
```

Comment:
Preferable=y we want the Snapshot to be immutable, but in this case it would just be a hassle to "roundtrip" through `Profile` in `ProfileStore`'s function `changeProfileSnapshot:to`, thus we allow this, but still make the setter private, having to go through `changeCurrentToMainnetIfNeeded`.